client/signup/steps/user/index.jsx

```diff
@@ -496,15 +505,53 @@ export class UserStep extends Component {
 		}
 
 		return (
-			<StepWrapper
-				flowName={ this.props.flowName }
-				stepName={ this.props.stepName }
-				headerText={ this.getHeaderText() }
-				subHeaderText={ this.getSubHeaderText() }
-				positionInFlow={ this.props.positionInFlow }
-				fallbackHeaderText={ this.props.translate( 'Create your account.' ) }
-				stepContent={ this.renderSignupForm() }
-			/>
+			<ProvideExperimentData
+				name="registration_social_login_first_on_mobile_v2"
+				options={ {
+					isEligible: true || ( isMobile() && this.props.flowName === 'onboarding' ),
```

Comment:
Should the `true` condition be changed here?

Reply:
Good catch!